pkgs/by-name/mu/muse-sounds-manager/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoPatchelfHook,
+  dpkg,
+  fontconfig,
+  zlib,
+  icu,
+  libX11,
+  libXext,
+  libXi,
+  libXrandr,
+  libICE,
+  libSM,
+  openssl,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "muse-sounds-manager";
+  version = "1.1.0.587";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20240826143936/https://muse-cdn.com/Muse_Sounds_Manager_Beta.deb";
+    hash = "sha256-wzZAIjme1cv8+jMLiKT7kUQvCb+UhsvOnLDV4hCL3hw=";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    dpkg
+  ];
+
+  buildInputs = [
+    fontconfig
+    stdenv.cc.cc
+    zlib
+  ] ++ runtimeDependencies;
+
+  runtimeDependencies = map lib.getLib [
+    icu
+    libX11
+    libXext
+    libXi
+    libXrandr
+    libICE
+    libSM
+    openssl
+  ];
+
+  unpackPhase = "dpkg -x $src .";
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out
+    mv usr/* opt $out/
+    substituteInPlace $out/bin/muse-sounds-manager --replace-fail /opt/ $out/opt/
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Manage MuseScore Libraries";
+    homepage = "https://musescore.org/";
+    license = licenses.unfree;
+    maintainers = with maintainers; [ orivej ];
+    platforms = platforms.linux;
```

Comment:
I think also, only `x86_64` is supported?

Reply:
Good catch!